amulet/sentry.py

```diff
@@ -120,10 +120,11 @@ def upload_scripts(self):
 
         # copy one at a time b/c `juju scp -r` doesn't work (currently)
         for f in glob.glob(os.path.join(source, '*')):
-            cmd = "juju scp {} {}:{}".format(
-                os.path.join(source, f),
-                self.info['unit_name'], dest)
-            subprocess.check_call(cmd.split())
+            src = os.path.join(source, f)
+            if os.path.isfile(src):
```

Comment:
There are four files that this needs to pick up: https://github.com/juju/amulet/tree/master/amulet/unit-scripts/amulet

It looks like this is a side-effect of [7adbf808](https://github.com/juju/amulet/commit/7adbf808497e1ae9cab7f304af3d478b01873de8#diff-c20575a530a26c0d438b922ba73613ebL117) where you relaxed the glob specifically to pick up the `.sh` file, @tvansteenburgh.

@kwmonroe We could use `-r` but we don't actually *want* to pick up the `__pycache__` (or any other) directories.

I think this is the cleanest solution.  We could change the glob to `*.[ps][yh]` but that seems a bit ugly.